azkaban-common/src/main/java/azkaban/utils/DirectoryFlowLoader.java

```diff
@@ -60,8 +67,9 @@
   private Set<String> errors;
   private Set<String> duplicateJobs;
 
-  public DirectoryFlowLoader(Logger logger) {
+  public DirectoryFlowLoader(Props props, Logger logger) {
```

Comment:
This may break existing code because a change in the constructor signature.  Can we overload the constructor?  Whether we use the props, needs to guard against props being null.